src/Uno.Extensions.Authentication.Oidc/OidcAuthenticationSettings.cs

```diff
@@ -6,4 +6,6 @@ namespace Uno.Extensions.Authentication.Oidc;
 internal record OidcAuthenticationSettings
 {
 	public OidcClientOptions? Options { get; init; }
+
+	public bool AutoRedirectUri { get; init; } = PlatformHelper.IsWebAssembly;
```

Comment:
The only problem I see with that is the impossibility to set this in the json appsettings file.

Also I set it to Wasm only for backward compatibility, but it would make sense to activate it everywhere.